src/main/java/hek/models/EventType.java

```diff
@@ -27,6 +31,10 @@ public String toQualifiedString() {
         return Arrays.asList(AR, SG, FL, CH);
     }
 
+    /**
+     * returns even types as List in Arrays class.
```

Comment:
Typo here and it can start with upper case, this will be a whole sentence when it is rendered.

This returns List of Solar Event Types (not events)